les/serverpool.go

```diff
@@ -198,6 +198,7 @@ func (pool *serverPool) discoverNodes() {
 		if err != nil {
 			continue
 		}
+		clientDiscoveredNodesCounter.Inc(1)
```

Comment:
I mentioned when we were chatting that it would be good to know the number of _new_ discovered nodes in addition to this, which is every node returned by discovery. It looks like [findOrNewNode](https://github.com/celo-org/celo-blockchain/pull/1129/files#diff-507764eec3e28419d12153b0d1fee8cdR442) provides the logic that determines if a node is new, and this might be worth hooking in to.

Reply:
we chatted offline a decided to better analyze serverpool before adding more metrics